modules/programs/mc.nix

```
@@ -0,0 +1,103 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.mc;
+  type = (pkgs.formats.ini { }).type;
+in
+{
+  options.programs.mc = {
+    enable = lib.mkEnableOption "Midnight Commander";
+
+    package = lib.mkPackageOption pkgs "mc" { nullable = true; };
+
+    settings = lib.mkOption {
+      inherit type;
+      default = { };
+      description = "Settings for `mc/ini` file. Any missing settings will fall back to the system default.";
```

```suggestion
      description = ''
        Settings for `mc/ini` file.
      
        Any missing settings will fall back to the system default.
       '';
```

Appreciate your patience. It would be nicer for legibility in docs if we used lines and used some newlines to separate thoughts for all these options.

---

Done